src/css/components/_captions-settings.scss

```diff
@@ -26,30 +26,28 @@
   .vjs-text-track-settings .vjs-modal-dialog-content {
     display: grid;
     grid-template-columns: 1fr 1fr;
-    grid-template-rows: 1fr auto;
+    grid-template-rows: 1fr;
+    min-height: 23em;
   }
 
-  .vjs-text-track-settings .vjs-track-settings-colors {
-    display: block;
-    grid-column: 1;
-    grid-row: 1;
+  .vjs-text-track-settings .vjs-track-settings-controls {
+    grid-column: 1 / -1;
   }
 
-  .vjs-text-track-settings .vjs-track-settings-font {
-    grid-column: 2;
-    grid-row: 1;
+  // 1 column for small players
+  .vjs-layout-small .vjs-text-track-settings .vjs-modal-dialog-content ,
+  .vjs-layout-x-small .vjs-text-track-settings .vjs-modal-dialog-content,
+  .vjs-layout-tiny .vjs-text-track-settings .vjs-modal-dialog-content {
+    grid-template-columns: 1fr;
+    min-height: 34.2em;
```

Comment:
We have to update the `min-height` here so that controls never touch the bottom of the modal